its/projects/ReproAzureFunctions/NuGet.Config

```diff
@@ -2,8 +2,14 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
-= </packageSources>
+    <add key="Repox" value="https://repox.jfrog.io/artifactory/api/nuget/v3/nuget/index.json" protocolVersion="3" />
```

Comment:
Why do we even need this file? I don't see anything new that it would add on top of the one from the root.

Reply:
The ITs are executed in a different folder (Temp) so the file from the root is not vible.

Reply:
I've tried to remove them without success. It seems there is a fallback configuration on the machine that causes some troubles: 
```
2024-01-26T09:47:55.5876876Z 09:47:55.524 INFO    Retrying 'FindPackagesByIdAsyncCore' for source '[https://repox.jfrog.io/artifactory/api/nuget/sonarsource-nuget-releases/FindPackagesById()?id='Microsoft.Azure.Functions.Analyzers'&semVerLevel=2.0.0](https://repox.jfrog.io/artifactory/api/nuget/sonarsource-nuget-releases/FindPackagesById()?id=%27Microsoft.Azure.Functions.Analyzers%27&semVerLevel=2.0.0)'.
2024-01-26T09:47:55.5877296Z 09:47:55.525 INFO    Response status code does not indicate success: 500.
2024-01-26T09:47:55.5877798Z 09:47:55.530 INFO    Retrying 'FindPackagesByIdAsyncCore' for source '[https://repox.jfrog.io/artifactory/api/nuget/sonarsource-nuget-releases/FindPackagesById()?id='Microsoft.Azure.WebJobs.Extensions'&semVerLevel=2.0.0](https://repox.jfrog.io/artifactory/api/nuget/sonarsource-nuget-releases/FindPackagesById()?id=%27Microsoft.Azure.WebJobs.Extensions%27&semVerLevel=2.0.0)'.
2024-01-26T09:47:55.5878212Z 09:47:55.530 INFO    Response status code does not indicate success: 500.
2024-01-26T09:47:55.5878835Z 09:47:55.531 INFO    Retrying 'FindPackagesByIdAsyncCore' for source '[https://repox.jfrog.io/artifactory/api/nuget/nuget/FindPackagesById()?id='Microsoft.Azure.Functions.Analyzers'&semVerLevel=2.0.0](https://repox.jfrog.io/artifactory/api/nuget/nuget/FindPackagesById()?id=%27Microsoft.Azure.Functions.Analyzers%27&semVerLevel=2.0.0)'.
2024-01-26T09:47:55.5879278Z 09:47:55.531 INFO    Response status code does not indicate success: 500.
2024-01-26T09:47:55.5879953Z 09:47:55.537 INFO    Retrying 'FindPackagesByIdAsyncCore' for source '[https://repox.jfrog.io/artifactory/api/nuget/nuget/FindPackagesById()?id='Microsoft.Azure.WebJobs.Extensions'&semVerLevel=2.0.0](https://repox.jfrog.io/artifactory/api/nuget/nuget/FindPackagesById()?id=%27Microsoft.Azure.WebJobs.Extensions%27&semVerLevel=2.0.0)'.
2024-01-26T09:47:55.5880598Z 09:47:55.537 INFO    Response status code does not indicate success: 500.
2024-01-26T09:47:55.5881112Z 09:47:55.545 INFO    Retrying 'FindPackagesByIdAsyncCore' for source '[https://repox.jfrog.io/artifactory/api/nuget/nuget/FindPackagesById()?id='Microsoft.Azure.WebJobs'&semVerLevel=2.0.0](https://repox.jfrog.io/artifactory/api/nuget/nuget/FindPackagesById()?id=%27Microsoft.Azure.WebJobs%27&semVerLevel=2.0.0)'.
2024-01-26T09:47:55.5881573Z 09:47:55.545 INFO    Response status code does not indicate success: 500.
2024-01-26T09:47:55.5882164Z 09:47:55.550 INFO    Retrying 'FindPackagesByIdAsyncCore' for source 'https://repox.jfrog.io/artifactory/api/nuget/sonarsource-nuget-releases/FindPackagesById()?
```
I would handle this separatelly.

Reply:
These sources are added here: https://github.com/SonarSource/re-ci-images/blob/master/ec2-images/ansible/roles/tools/tasks/nuget.yml#L46-L54

I think we should either fix or remove them.